lib/orchestrator.js

```diff
@@ -16,6 +16,9 @@ var TokenChecker = require('./token-checker');
 // Configuration file
 require('../config.js');
 
+// Pseudo-constants:
+var W3_ORG_PREFIX = /^https?:\/\/(www\.)?w3c?\.org/i;
```

Comment:
Ah ah, now that I am not around to grumble anymore, they are back! But, seriously, `http://w3c.org/`? And `http://WwW.w3C.OrG/`? Dude...